docs/source/software/basic_flight_controller.rst

```diff
@@ -1,3 +1,86 @@
 =======================
 Basic Flight Controller
 =======================
+
+The flight controller is responsible for passing high level position information to the px4. The Controller is written in C++ and designed to be extended for multiple use cases. To develop your own controller see the controller development page.
```

Comment:
Should we clarify "px4" vs "pixhawk/pixracer"? px4 is the flight software, which runs on different controllers, right?

Reply:
changed to "pixhawk/pixracer"